flytekit/models/literals.py

```diff
@@ -852,6 +852,52 @@ def from_flyte_idl(cls, pb2_object):
         )
 
 
+class LiteralOffloadedMetadata(_common.FlyteIdlEntity):
```

Comment:
Not your change, but a generator class for this could be useful

Reply:
do you mean like a utility function to help in tests?